doc/tutorials/Readme.rst

```diff
@@ -16,6 +16,7 @@ physical systems. Currently, the following tutorials are available:
 * :file:`08-visualization`: Using the online visualizers of ESPResSo
 * :file:`11-ferrofluid`: Modelling a monolayer ferrofluid system
 * :file:`12-constant_pH`: Modelling the titration of a weak acid using the constant pH method
+* :file:`13-charged_rod`: Modelling counterion condensation on a charged rod
```

Comment:
Numbers mismatch. But do we really want to delete the crystal stuff?

Reply:
A few comments on the crystal tutorial:
* Minor point(?): for @christophlohrmann and myself it was not clear what the physical motivation of such a simulation would be: For nanocapacitors/elechtrochemistry you need a constant charge or constant E-field whereas we employ a constant D-Field and the cystallization itself is not so fascinating IMHO.
* More important: The tutorial focuses a lot on physical constants/units and how to convert them to ESPResSo, but there are a lot of inconsistencies that we could not resolve. For example, the mass density of the ionic liquid was chosen to match experimental numbers, but in the script the electrode mass was included??
* Major point: Througout the tutorial we consider an ionic liquid close to its coexistence with the crystal, i.e. correlation times explode which is not really useful for an interactive tutorial (I agree though that it's something we should/could have somewhere [where?]). The dilute electrolyte is much nicer in this respect.
Again, I'm not insisting on removing the crystal tutorial, but to make it convincing IMO there is much more work needed than in the present PR for the dilute electrolyte (which also offers physical questions/interpretations in much shorter simulation times).